docs/index.html

```diff
@@ -1,73 +1,9 @@
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-<title>生成されたドキュメント(タイトルなし)</title>
-<script type="text/javascript">
-    tmpTargetPage = "" + window.location.search;
-    if (tmpTargetPage != "" && tmpTargetPage != "undefined")
-        tmpTargetPage = tmpTargetPage.substring(1);
-    if (tmpTargetPage.indexOf(":") != -1 || (tmpTargetPage != "" && !validURL(tmpTargetPage)))
-        tmpTargetPage = "undefined";
-    targetPage = tmpTargetPage;
-    function validURL(url) {
-        try {
-            url = decodeURIComponent(url);
-        }
-        catch (error) {
-            return false;
-        }
-        var pos = url.indexOf(".html");
-        if (pos == -1 || pos != url.length - 5)
-            return false;
-        var allowNumber = false;
-        var allowSep = false;
-        var seenDot = false;
-        for (var i = 0; i < url.length - 5; i++) {
-            var ch = url.charAt(i);
-            if ('a' <= ch && ch <= 'z' ||
-                    'A' <= ch && ch <= 'Z' ||
-                    ch == '$' ||
-                    ch == '_' ||
-                    ch.charCodeAt(0) > 127) {
-                allowNumber = true;
-                allowSep = true;
-            } else if ('0' <= ch && ch <= '9'
-                    || ch == '-') {
-                if (!allowNumber)
-                     return false;
-            } else if (ch == '/' || ch == '.') {
-                if (!allowSep)
-                    return false;
-                allowNumber = false;
-                allowSep = false;
-                if (ch == '.')
-                     seenDot = true;
-                if (ch == '/' && seenDot)
-                     return false;
-            } else {
-                return false;
-            }
-        }
-        return true;
-    }
-    function loadFrames() {
-        if (targetPage != "" && targetPage != "undefined")
-             top.classFrame.location = top.targetPage;
-    }
-</script>
-</head>
-<frameset cols="20%,80%" title="Documentation frame" onload="top.loadFrames()">
-<frame src="allclasses-frame.html" name="packageFrame" title="すべてのクラスおよびインタフェース(非staticのネストされた型を除く)">
-<frame src="com/nifcloud/mbaas/core/package-summary.html" name="classFrame" title="パッケージ、クラスおよびインタフェースの説明" scrolling="yes">
-<noframes>
-<noscript>
-<div>ブラウザのJavaScriptが無効になっています。</div>
-</noscript>
-<h2>フレーム関連のアラート</h2>
-<p>このドキュメントはフレーム機能を使用して表示するように作成されています。フレームを表示できないWebクライアントの場合にこのメッセージが表示されます。<a href="com/nifcloud/mbaas/core/package-summary.html">フレームに対応していないバージョン</a>にリンクします。</p>
-</noframes>
-</frameset>
-</html>
+<!DOCTYPE html>
+<title>Redirecting to Reference</title>
+<meta http-equiv="refresh" content="0; URL=https://javadoc.dev/online/api/com.nifcloud.mbaas/ncmb_android/4.1.0/index.html">
+<link rel="canonical" href="https://javadoc.dev/online/api/com.nifcloud.mbaas/ncmb_android/4.1.0/index.html">
+<body>
+    <p>ニフクラ mobile backendのAndroid SDK リファレンスにリダイレクトします。</p>
```

Comment:
日本語的はもしかしたらこちらの方が良いかと思いますが、いかがでしょうか。
```suggestion
    <p>ニフクラ mobile backend Android SDK リファレンスにリダイレクトします。</p>
```